### ISSUE TITLE
[css-shapes-2] Allow `position` for the initial coordinates of `shape()`

### ISSUE THREAD

**Issue by noamr** [w3c/csswg-drafts#11358]
Currently in `shape()`, the path can `move to` a `position`, but it can only start from a `<<coordinate-pair>>`. This seems like an unnecessary restriction.

**Comment by noamr** [comment]
@smfr does this make sense? Happy to quickly tackle this as an async resolution if so.

**Comment by smfr** [comment]
Yes, this should be a `<position>`; I implemented it as such in WebKit.

**Comment by noamr** [comment]
@astearns let's file an async resolution for this one?

**Comment by astearns** [comment]
The CSSWG will automatically accept this resolution one week from now if no objections are raised here. Anyone can add an emoji to this comment to express support. If you do not support this resolution, please add a new comment.

Proposed Resolution: Allow `position` for the initial coordinates of `shape()`

**Comment by astearns** [comment]
RESOLVED: Allow `position` for the initial coordinates of `shape()`
